src/targets/maven.ts

```diff
@@ -21,6 +21,9 @@ const GRADLE_PROPERTIES_FILENAME = 'gradle.properties';
  * https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_environment_variables
  */
 const DEFAULT_GRADLE_USER_HOME = join(homedir(), '.gradle');
+export const POM_DEFAULT_FILENAME = 'pom-default.xml';
+const POM_FILE_EXTNAME = '.xml'; // Must include the leading `.`
```

Comment:
nit: name is redundant
```suggestion
const POM_FILE_EXT = '.xml'; // Must include the leading `.`
```